test/unit/algorithm_toolkit/test_hyperparameter_validation.py

```diff
@@ -187,3 +187,72 @@ def test_simple_tuple(self):
 
         result = hyperparameters.validate({"tuple_": "(1,0,-1)"})
         self.assertEqual(result["tuple_"], (1, 0, -1))
+
+    def test_tuple_alias(self):
+        hyperparameters = hpv.Hyperparameters(hpv.TupleHyperparameter(name="tuple_",
+                                                                           range=[-1, 0, 1],
+                                                                           required=False))
+        hyperparameters._declare_alias("tuple_", "tup_")
+
+        with self.assertRaises(exc.UserError):
+            hyperparameters.validate({"tup_": "(1,0,-1,2)"})
```

Comment:
is this test truly required for this code change? Why do we need to test the range parameter?